docs/DeveloperGuide.adoc

```diff
@@ -334,21 +354,69 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given category is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. System shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+* 3b. The given index is invalid.
++
+[none]
+** 3b1. System shows an error message.
++
+Use case resumes at step 2.
+
+
+[discrete]
+=== Use case: Edit Flashcard
+
+*MSS*
+
+1.  User requests to list all flashcards in a category
+2.  System shows a list of flashcards with index
+3.  User requests to edit a flashcard in the list by providing category and index
+4.  System provides a pop-up for user to enter changes
+5.  User enters changes
```

Comment:
Possible extension: after splitting step 5 into 'user enters changes' and 'user submits changes', if they close the app before submitting changes, will the data be lost?

Reply:
You can also consider adding an option to abort changes before submitting